pkg/sys/mock/runner.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright © 2022 - 2025 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mock
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/suse/elemental/v3/pkg/sys/log"
+)
+
+type FakeRunner struct {
```

Comment:
Can we drop all "Test" and "Fake" prefixes from the type names? They're now all coming from `mock` so it is clear they're such.